src/Tooltip.js

```diff
@@ -163,27 +164,101 @@ export default class Tooltip extends BaseClass {
 
     divElement("footer");
 
+    divElement("arrow");
+
     enter.call(boxStyles);
 
     const t = transition().duration(this._duration);
 
     update
-      .attr("id", (d, i) => `d3plus-tooltip-${this._id(d, i)}`)
+      .attr("id", (d, i) => `d3plus-tooltip-${d ? this._id(d, i) : ""}`)
       .transition(t)
-        .style(`${this._prefix}transform`, "scale(1)")
+        .style("opacity", 1)
         .call(boxStyles);
 
     tooltips.exit()
       .transition(t)
-        .style(`${this._prefix}transform`, "scale(0)")
-        .remove();
+        .style("opacity", 0)
+      .remove();
+
+    for (let i = 0; i < this.data.length; i++) {
+      if (that._data[i]) {
+        const tooltip = document.getElementById(`${this._className}-${that._id(that._data[i], i)}`);
+        const arrow = document.getElementById(`d3plus-tooltip-arrow-${that._id(that._data[i], i)}`);
+        const arrowHeight = arrow.getBoundingClientRect().height;
+        arrow.style.bottom = `-${arrowHeight + 1}px`;
+
+        const arrowOffset = arrowHeight / 4;
+
+        const referenceObject = that._position() instanceof Element ? that._position() : {
+          clientWidth: 0,
+          clientHeight: 0,
+          getBoundingClientRect: () => ({
+            top: that._position(that._data[i])[1] - arrowOffset,
+            right: that._position(that._data[i])[0] - arrowOffset,
+            bottom: that._position(that._data[i])[1] - arrowOffset,
+            left: that._position(that._data[i])[0] - arrowOffset,
+            width: 0,
+            height: 0
+          })
+        };
+
+        new Popper(referenceObject, tooltip, {
+          placement: "top",
+          modifiers: {
+            arrow: {
+              element: arrow
+            },
+            flip: {
+              enabled: false
+            },
+            offset: {offset: `0,${that._offset()}`}
+          },
+          onCreate({instance}) {
+            document.onmousemove = () => {
+              instance.scheduleUpdate();
+            };
+          }
+        });
+      }
+    }
 
     if (callback) setTimeout(callback, this._duration + 100);
 
     return this;
 
   }
 
+  /**
+   @memberof Tooltip
+   @desc If *value* is specified, sets the arrow accessor to the specified function or string and returns this generator. If *value* is not specified, returns the current arrow accessor.
```

Comment:
Can the description be more like: "Sets the inner HTML content of the arrow element, which by default is empty."